GradientColorPickerTest/IntegrationTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Drawing;
+using Messerli.GradientColorPicker;
+using Xunit;
+
+namespace Messerli.GradientColorPickerTest
+{
+    public class IntegrationTest
+    {
+        private const double GoodPasswordEntropy = 40.0;
```

Comment:
Restrict the scope of these variables

Reply:
Addressed in: https://github.com/messerli-informatik-ag/gradient-color-picker/pull/4/commits/f437c7f2a47dce035e245c81c81acd0ecea615f3